### ISSUE TITLE
Implement Text Reflow

### ISSUE THREAD

**Issue by snelltheta** [SufficientlySecure/document-viewer#127]
Should this be left to the muPDF people, or should it be here?
Anyways, text reflow has to be one of the most important features for a pdf-capable reader on a small device. 

Any ideas how this could be implemented?

**Comment by tuxor1337** [comment]
Yes, this is probably best implemented as part of the muPDF library since it manages layouting of running text already in its epub capability. Document Viewer itself should stick to providing a user interface for the layouting libraries and should only be concerned with minor filtering tasks.

**Comment by mendelp** [comment]
This is implemented in MuPDF, their android sample app has a reflow option. I think you just need to expose it in the interface.

**Comment by ericwa** [comment]
I would do sooner if it was a simple change, but mupdf's reflow support would be awkward to integrate with DV because it just returns HTML, whereas DV's mupdf backend is just set up for rendering images.

IMHO, returning HTML to the app is not a proper way of doing reflow, since you lose all images and embedded fonts.. the reflowing should really be done inside mupdf for it to work decently.

**Comment by ildar** [comment]
Any new ideas here?
